cognite_toolkit/_cdf_tk/constants.py

```diff
@@ -77,6 +77,49 @@
 HINT_LEAD_TEXT_LEN = 5
 EnvType: TypeAlias = Literal["dev", "test", "staging", "qa", "prod"]
 USE_SENTRY = "pytest" not in sys.modules and os.environ.get("SENTRY_ENABLED", "true").lower() == "true"
+SPACE_FORMAT_PATTERN = r"^[a-zA-Z][a-zA-Z0-9_-]{0,41}[a-zA-Z0-9]?$"
```

Comment:
This space format pattern looks buggy, is this the official definition? I would guess the intention of this pattern is to make sure the string does not end with `_` or `-`, but that is not how it is written.

The container pattern looks sane, as it has the right parentheses